Sources/Exporters/OpenTelemetryProtocolCommon/proto/metrics.pb.swift

```diff
@@ -997,6 +997,10 @@ public struct Opentelemetry_Proto_Metrics_V1_SummaryDataPoint {
 
   public var unknownFields = SwiftProtobuf.UnknownStorage()
 
+  /// (Optional) List of exemplars collected from
+  /// measurements that were used to form the data point
+  public var exemplars: [Opentelemetry_Proto_Metrics_V1_Exemplar] = []
```

Comment:
we shouldn't edit this file. The summary data point does not have exemplars https://github.com/open-telemetry/opentelemetry-proto/blob/v1.5.0/opentelemetry/proto/metrics/v1/metrics.proto#L616

Reply:
Remove this change.